ui/src/api/auth-api.ts

```diff
@@ -0,0 +1,11 @@
+import RestClient from './rest-client';
+
+export default class AuthApi {
+  static createUser(url: string, data: { email: string; password: string }) {
+    return RestClient.post(url, data);
+  }
+
+  static fetchUser(url: string) {
```

Comment:
I don't like that we pass the url as a param here. I think it should be specified on a service level